tasks/ma.yml

```diff
@@ -1,6 +1,6 @@
 MA-scrape:
   image: openstates/scrapers
-  entrypoint: "poetry run os-update ma bills sort=latest page_limit=20"
+  entrypoint: "poetry run os-update ma bills sort=latest page_limit=2 --realtime"
```

Comment:
Just curious why are you changing this from page_limit=20 to page_limit=2 ?

Reply:
OOps, no I'm not good catch